code/src/components/Signup.css

```diff
@@ -0,0 +1,216 @@
+.Signup {
```

Comment:
The CSS styling for the form elements, such as radio buttons and input fields, is consistent and visually appealing. The radio buttons have a custom appearance, and the input fields have a clear focus state, which contributes to a good user experience.